nvchecker/gen-config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+cd "${0%/*}/.."
+ROOT="$(pwd)"
+
+CFG=nvchecker/archlinux-proaudio.toml
+VER=nvchecker/old_ver.json
+cat > $CFG << __EOF__
+[__config__]
+oldver = "old_ver.json"
+newver = "new_ver.json"
+keyfile = "keyfile.toml"
+__EOF__
+
+VER_TEMP="$(mktemp)"
+for f in packages/*/.nvchecker.toml; do
+  echo "" >> $CFG
+  cat $f >> $CFG
+  pkgdir=$(dirname $f)
+  pkgbase=$(basename $pkgdir)
+  pkgver=$(. $pkgdir/PKGBUILD; echo $pkgver)
+  printf '{"%s": "%s"}\n' $pkgbase $pkgver >> "$VER_TEMP"
+done
+jq -s add "$VER_TEMP" > $VER
```

Comment:
Please add a note to the readme that `jq` needs to be installed to run the `run-nvchecker.cron` script.

Reply:
46f20ae7ddeedb7f89c7390be9791f93cc1e6bc9